game/gui.py

```diff
@@ -77,8 +94,9 @@ def draw_block(self, x: int, y: int, color) -> None:
             rect=Rect(x * BLOCK_SIZE, y * BLOCK_SIZE, BLOCK_SIZE, BLOCK_SIZE)
         )
 
-    def draw_title(self):
-        self.screen.blit(self.graphics['title'], (TITLE_X, TITLE_Y));
+    def draw_game_images(self):
+        for img in self.active_images:
+            self.screen.blit(img[0], img[1])
```

Comment:
what's img[0] and img[1]? can you add some comments or documentation

Reply:
Yes